config/development.js

```diff
@@ -10,7 +10,8 @@ exports.config = {
     },
     jsonPlaceHolderApi: {
       baseUrl: process.env.JSON_PLACE_HOLDER_API_BASE_URL
-    }
+    },
+    availableDomains: process.env.AVAILABLE_DOMAINS
```

Comment:
don't forget to update the `env.example` file, please

Reply:
Updated! :)